server/chat.ts

```diff
@@ -1657,10 +1657,10 @@ export const Chat = new class {
 			if (typeof entry !== 'function') continue;
 
 			const handlerCode = entry.toString();
-			entry.requiresRoom = /\bthis\.requiresRoom\(/.test(handlerCode);
-			entry.hasRoomPermissions = /\bthis\.can\([^,)\n]*, [^,)\n]*,/.test(handlerCode);
-			entry.broadcastable = /\bthis\.(?:canBroadcast|runBroadcast)\(/.test(handlerCode);
-			entry.isPrivate = /\bthis\.(?:privatelyCan|commandDoesNotExist)\(/.test(handlerCode);
+			entry.requiresRoom = /\bthis\.requireRoom\(/.test(handlerCode);
+			entry.hasRoomPermissions = /\bthis\.checkCan\([^,)\n]*, [^,)\n]*,/.test(handlerCode);
```

Comment:
This should probably include the original names as well, since they're still usable.